app/package.json

```diff
@@ -48,16 +48,22 @@
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-jest": "^26.5.3",
     "eslint-plugin-storybook": "^0.5.12",
+    "i18next": "^21.8.13",
+    "i18next-browser-languagedetector": "^6.1.4",
+    "i18next-http-backend": "^1.4.1",
     "jest": "^28.1.0",
     "jest-axe": "^6.0.0",
     "jest-cli": "^28.1.0",
     "jest-environment-jsdom": "^28.1.0",
+    "next-i18next": "^11.0.0",
     "postcss": "^8.4.14",
     "postcss-loader": "^7.0.0",
     "postcss-preset-env": "^7.7.2",
     "prettier": "^2.6.2",
+    "react-i18next": "^11.18.0",
```

Comment:
nit: I think `i18next`, `next-i18next`, and `react-i18next` would be `dependencies` rather than `devDependencies` since those would end up production bundles.